src/lib/apollo/cache.ts

```diff
@@ -80,6 +82,15 @@ export const createCache = () =>
           people: paginationFieldPolicy,
           tasks: paginationFieldPolicy,
           userNotifications: paginationFieldPolicy,
+          // When loading a user option, look it up from the cache by its key
+          userOption: {
```

Comment:
This makes it so that if some other component loaded all the user options with the `userOptions` query, the `userOption` (singular) query can pull the option out of the cache by its key instead of having to wait for it to load.